asreview/project.py

```diff
@@ -58,46 +60,112 @@ class ProjectExistsError(Exception):
 class ProjectNotFoundError(Exception):
     pass
 
+# !=============================================
+# @TODO: I HATE THE FACT THAT I AM TALKING ABOUT 
+# Users in this module, but I need a quick fix
+# for now
+# !=============================================
 
-def get_project_path(project_id, asreview_dir=None):
+def get_project_path(project_id, user=None, asreview_dir=None):
     """Get the project directory.
 
     Arguments
     ---------
     project_id: str
         The id of the current project.
+    user: User
+        User account object.
     """
 
     if asreview_dir is None:
         asreview_dir = asreview_path()
 
-    return Path(asreview_dir, project_id)
+    if isinstance(user, User) and isinstance(user.id, int):
```

Comment:
Twice the same if statement. You can remove the outer layer I think.

Reply:
I did.

Question: in the authenticated version we may or may not have collaboration. If a collaborator needs to work on a project, the project_id must come from owner id plus project id. In the current situation I have an else statement in case all fails. That is not supposed to happen. Shall I remove the else clause? The else clause is a fail-safe.

Reply:
If it's not wanted behavior that the else clause is reached maybe raising an error there is better?